redash/query_runner/dynamodb_sql.py

```diff
@@ -102,8 +102,10 @@ def run_query(self, query, user):
         try:
             engine = self._connect()
 
-            result = engine.execute(query if str(
-                query).endswith(';') else str(query) + ';')
+            if not query.endswith(';'):
```

Comment:
This logic isn't perfect but it's the same as we had before just without the unicode issue.